docs/products/postgresql/howto/pagila.rst

```diff
@@ -0,0 +1,251 @@
+Pagila – a PostgreSQL sample database
+=====================================
+
+Aiven provides a sample database you can import in your Aiven for PostgreSQL service. This page covers information about the database and the procedure to get it up and running.
```

Comment:
I kept the link on the original place since it makes sense to talk about it a bit before linking to it. On the first paragraph, I just mention what the article is about.